code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -940,6 +940,20 @@ datum/reagent/medicine/styptic_powder/overdose_start(mob/living/M)
 	description = "Reacts with neural tissue, helping reform damaged connections. Can cure minor traumas."
 	color = "#EEFF8F"
 
+/datum/reagent/neurine/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	var/obj/item/organ/brain/B = M.getorganslot(ORGAN_SLOT_BRAIN)
+	if(!B)
+		return
+	if(method == INJECT)
```

Comment:
All good now, just make this an early return found before the getorganslot line.

Reply:
How can I do (!B) without B?

Reply:
Oh